src/components/responsive-hero-image.js

```diff
@@ -18,9 +19,28 @@ import omit from '@babel/runtime/helpers/objectWithoutProperties';
  * @category Images
  */
 export function ResponsiveHeroImage(props) {
-  return React.createElement(ResponsiveImage, Object.assign({
-    slug: props.story["hero-image-s3-key"],
-    metadata: props.story["hero-image-metadata"],
-    alt: props.story["headline"]
-  }, omit(props, ['story'])));
+  const { story = {} } = props;
```

Comment:
why do we need a fallback for story? also, it would be cleaner to destructure props in `export function ResponsiveHeroImage(props)` the  than doing it here